scripts/APIv4.js

```diff
@@ -51,18 +54,26 @@ const ensureDir = (dirPath) => fs.mkdir(dirPath, { recursive: true }).
 const processEntries = async (files) => {
   const entries = {};
 
-  await Promise.all(files.map(async (file) => {
-    const data = await readJSONFile(file);
-    const entry = data[Object.keys(data)[0]];
+  await Promise.all(
+    files.map(async (file) => {
+      const data = await readJSONFile(file);
+      const entry = data[Object.keys(data)[0]];
 
-    // Add the main domain entry
-    entries[entry.domain] = entry;
+      // Rename contact.twitter to contact.x if it exists
+      if (entry.contact && entry.contact.twitter) {
+        entry.contact.x = entry.contact.twitter;
+        delete entry.contact.twitter;
+      }
```

Comment:
Code that renamed twitter to x.
All other changes in APIv4.js are reformatting.